rlkit/core/rl_algorithm.py

```diff
@@ -224,7 +206,7 @@ def train(self):
                 self._n_train_steps_total += 1
             gt.stamp('train')
 
-            #self.training_mode(False)
+            self.training_mode(False)
```

Comment:
Do we actually use this flag now?

Reply:
ah I meant to make this a separate commit to highlight it....it breaks the LSTM for reasons unknown to me but I think it's unsafe to omit it unless we have to, and I don't think `master` needs to support the LSTM